routes/api/playlists.ts

```diff
@@ -9,7 +9,7 @@ const playlist = require('../../models/playlist');
 
 /* POST playlist. */
 router.post('/', oauth.authenticate({scope:"Indexer"}), function(req, res, next) {
-    playlist.savePlaylist(req.body.name, req.body.songs, res.locals.oauth.token.user.id, req.body.playlisttype, req.body.startdatetime, req.body.enddatetime, function(error, result){
+    playlist.saveOrUpdatePlaylist(req.body.name, req.body.songs, res.locals.oauth.token.user.id, req.body.playlisttype, req.body.startdatetime, req.body.enddatetime, function(error, result){
```

Comment:
Mogelijk handiger de methode gewoon te copyen, of savePlaylist als *post* en saveOrUpdate als *PUT*. 

Zie de REST standaard: http://www.restapitutorial.com/lessons/httpmethods.html
'PUT is most-often utilized for **update** capabilities, PUT-ing to a known resource URI with the request body containing the newly-updated representation of the original resource.

However, PUT can also be used to create a resource in the case where the resource ID is chosen by the client instead of by the server. In other words, if the PUT is to a URI that contains the value of a non-existent resource ID. Again, the request body contains a resource representation. Many feel this is convoluted and confusing. Consequently, this method of creation should be used sparingly, if at all.'
-- in principe niet wenselijk dus, maar zeer toepasselijk bij een create or update (wat in principe van zichzelf normaliter niet wenselijk is).